sdk/servicebus/Microsoft.Azure.Management.ServiceBus/src/Generated/DisasterRecoveryConfigsOperations.cs

```diff
@@ -1431,7 +1434,7 @@ public async Task<AzureOperationResponse> BreakPairingWithHttpMessagesAsync(stri
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<AzureOperationResponse> FailOverWithHttpMessagesAsync(string resourceGroupName, string namespaceName, string alias, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse> FailOverWithHttpMessagesAsync(string resourceGroupName, string namespaceName, string alias, FailoverProperties parameters = default(FailoverProperties), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
We cannot add new optional parameters in the middle because this is a breaking change. Could you please change your swagger accordingly to make sure you add this new parameter at the end of the parameter list?